docs/en/query_language/create.md

```diff
@@ -119,6 +119,7 @@ Supported compression algorithms:
 - `ZSTD[(level)]` — [ZSTD compression algorithm](https://en.wikipedia.org/wiki/Zstandard) with configurable `level`. Possible levels: [1, 22]. Default value: 1.
 - `Delta(delta_bytes)` — compression approach, when raw values are replaced with the difference of two neighbour values. Up to `delta_bytes` are used for storing delta value, so `delta_bytes` is a maximum size of raw values.
```

Comment:
Maybe indeed let's rename them from "compression approach" to "data preparation approach" like you did in Russian version? And explicitly say that such preprocessed sequences then often compress better using general purpose compression.